src/app/features/settings/components/settings-form/settings-form.component.html

```diff
@@ -0,0 +1,95 @@
+<form class="w-1/2" [formGroup]="form" (ngSubmit)="submit()">
+  <p class="text-blue-600 font-bold mt-4">General</p>
+  <p class="form-input-label">Avatar</p>
+
+  <custom-file-upload
+    accept=".jpg,.jpeg"
+    [url]="user().avatar"
+    (upload)="uploadFile($event)"
+  />
+
+  <label class="form-input-label" for="name">Name</label>
+  <input
+    class="form-input"
+    formControlName="name"
+    id="name"
+    placeholder="Name"
+    type="text"
+  />
+  @if (!form.controls['name'].untouched) {
```

Comment:
You are saying "if the input is **not untouched**" with double negation. It is hard to read. Why not to say "if input is touched"? It the same thing, but easier to understand

Reply:
Also, I guess just touching an input is not enough to show an error. It should also be invalid